source/features/rgh-linkify-features.tsx

```diff
@@ -64,6 +64,7 @@ void features.add(import.meta.url, {
 		pageDetect.isReleasesOrTags,
 		pageDetect.isCommitList,
 		pageDetect.isSingleCommit,
+		pageDetect.isRepoWiki,
```

Comment:
💖 I was going to ask for this! (Since a few days ago when wiki was opened, that is)